src/Behat/Config/Config.php

```diff
@@ -38,8 +51,47 @@ public function withProfile(Profile $profile): self
         return $this;
     }
 
+    public function withPreferredProfile(string $profileName): self
```

Comment:
There was no function to add this specific setting programmatically, so I added it